src/components/pages/Login/index.tsx

```diff
@@ -53,10 +52,9 @@ const Login = () => {
             }
         } else if(account.status === 404) {
             // TODO: Account not found! What shall be done then?
-            setIsLoading(false);
             setErrorMsg("Account not found!");
         }
-
+        if(account.status !== 0) setIsLoading(false);
```

Comment:
You can replace that line with the one below

```suggestion
        else setIsLoading(false);
```

Reply:
or just place `setIsLoading(false)` outside of the condition and remove the current reducing the loader hiding logic to just one line of code.

Like in the example below:
```suggestion
if(account.status === 404) {
               setErrorMsg("Account not found!");
}
setIsLoading(false);
```